pontoon/base/templates/widgets/multiple_choice.html

```diff
@@ -8,7 +8,7 @@
     </div>
     <ul>
       {% for user in choices %}
-      <li class="clearfix{% if user in extra %} contributor{% endif %}" data-id="{{ user.id }}" title="{{ user.first_name }}"><span class="arrow fa"></span>{{ user.email }}</li>
+      <li class="clearfix{% if user.email in extra %} contributor{% endif %}" data-id="{{ user.id }}" title="{{ user.first_name }}"><span class="arrow fa"></span>{{ user.email }}</li>
```

Comment:
That was actually a bug. For project permissions, we compared a user dict to user objects, so we never had any matches.